pkg/landscape/app/metadata-pull-hook.hoon

```diff
@@ -322,6 +325,7 @@
   =/  =associations:metadata
     (metadata-for-group:met resource)
   :_  this
+  :-  `card`[%give %fact ~[/nack] resource+!>(resource)]
```

Comment:
Do we ever not give the resource on `/nack` here? Seems like this could just be in the /lib/pull-hook.

Reply:
good point!